django_ninja_crudl/renderers.py

```
@@ -0,0 +1,30 @@
+        if isinstance(o, BaseModel):
+            return o.model_dump()
+        if isinstance(o, (Url, AnyUrl)):
+            return str(o)
+        if isinstance(o, (IPv4Address, IPv4Network, IPv6Address, IPv6Network)):
+            return str(o)
+        if isinstance(o, Enum):
+            return str(o)
+        return super().default(o)  # pyright: ignore [reportAny]
+
+
+class CrudlJSONRenderer(JSONRenderer):
```

Ruff (0.11.9)

Missing docstring in public class

(D101)